docs/manual/parameters/index.ipynb

```diff
@@ -601,8 +601,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Recall that Mojo can infer parameter values in a parametric function, based on\n",
-    "the parametric values attached to an argument value (see the `rsqrt[]()`\n",
+    "Recall that when a parametric function is called, Mojo can infer the parameter values.\n",
+    "In particular, it can use the parameter values attached to an argument value (see the `sqrt[]()`\n",
```

Comment:
"In particular" suggests that there are other ways to infer parameters. Maybe:
```suggestion
    "That is, it can use the parameter values attached to an argument value (see the `sqrt[]()`\n",
```

We need to add a section that defines parameter inference explicitly, but it's going to take a little more rewriting. Meanwhile, this is an improvement.